clippy_dev/src/release.rs

```diff
@@ -0,0 +1,27 @@
+use std::fmt::Write;
+use std::path::Path;
+
+use crate::utils::{UpdateMode, clippy_version, replace_region_in_file};
+
+const CARGO_TOML_FILES: [&str; 5] = [
```

Comment:
```suggestion
const CARGO_TOML_FILES: [&str; 4] = [
```

This should fix the CI error :)